client/me/purchases/manage-purchase/index.jsx

```diff
@@ -979,5 +986,7 @@ export default connect( ( state, props ) => {
 		relatedMonthlyPlanSlug,
 		relatedMonthlyPlanPrice,
 		isJetpackTemporarySite: purchase && isJetpackTemporarySitePurchase( purchase.domain ),
+		purchaseIsInAppPurchase: isInAppPurchase( state, purchase?.id ),
+		purchaseManagementIsLocked: isPurchaseManagementLocked( state, purchase?.id ),
```

Comment:
I think these two selectors are redundant and don't need to exist at all. You can instead do a direct `purchase?.isLocked` or `purchase?.isInAppPurchase` check.

Reply:
Ah yes, updated!